_posts/2017-08-14-fec-case-study.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How the FEC.gov team uses the Web Design Standards
+tags:
+- web design standards
+- design
+- case study
+- design
+---
+The U.S. Web Design Standards are currently being used on hundreds of government
+sites, with an audience of more than 59 million monthly users. In this eighth
+post in our series, we met with Jennifer Thibault, visual designer and
+innovation specialist, to talk about the Federal Election Commission (FEC)’s use
+of the Standards on [fec.gov][].
+
+**Standards team:** How did you find out about the U.S. Web Design Standards?
+
+**Jennifer Thibault:** We knew of the USWDS project early on since it was
+kicked off by a team inside 18F. But, we needed to get out a basic set of
+pages and content models on fec.gov [before][] the Standards team released
```

Comment:
@toolness What if we changed the link from "before" to "basic set of pages and content models"? I don't typically like one word links either.